com.unity.netcode.gameobjects/Tests/Editor/Transports/BatchedSendQueueTests.cs

```diff
@@ -8,7 +8,7 @@ namespace Unity.Netcode.EditorTests
 {
```

Comment:
I'm not 100% familiar with those tests, maybe they cover this case already, but with a new feature, shouldn't there be new tests as well?

Reply:
My memory of writing these tests is that they were pretty thorough and would cover all the new code paths. After looking at the tests, it seems my memory was wrong. I'll add some more tests.